backend/src/modules/idim-webservice/idim-webservice.service.ts

```diff
@@ -132,4 +142,109 @@ export class IdimWebserviceService {
             );
         }
     }
+
+    async verifyBceidUser(
+        userId: string,
+        requesterUserGuid: string,
+        requesterAccountTypeCode: string
+    ): Promise<HttpException | BCEIDUserResponse> {
+        if (!this.hasAllRequiredParams()) {
+            return new HttpException(
+                {
+                    error: 'Missing IDIM web service crednetials in the environment variables',
+                },
+                HttpStatus.INTERNAL_SERVER_ERROR
+            );
+        }
+
+        try {
+            const client = await this.getSoapClient();
+            // set xml schema parameters for the request
+            const requestData = {
+                accountDetailRequest: {
+                    onlineServiceId: this.idimWebServiceID,
+                    // who is sending the request
+                    requesterAccountTypeCode,
+                    requesterUserGuid,
+                    // who we search for, exact match userID
+                    userId,
+                    accountTypeCode: RequesterAccountTypeCode.Business,
```

Comment:
think we only search for business bceid, just hardcode it to be the business type